.rubocop_todo.yml

```diff
@@ -17,49 +17,49 @@ Lint/AmbiguousRegexpLiteral:
 Lint/InheritException:
   Enabled: false
 
-# Offense count: 20
+# Offense count: 22
 Lint/ParenthesesAsGroupedExpression:
   Enabled: false
 
-# Offense count: 32
+# Offense count: 31
 Lint/RescueException:
   Enabled: false
 
-# Offense count: 48
+# Offense count: 51
 # Cop supports --auto-correct.
 # Configuration parameters: AllowUnusedKeywordArguments, IgnoreEmptyMethods.
 Lint/UnusedMethodArgument:
   Enabled: false
 
-# Offense count: 143
+# Offense count: 160
 # Configuration parameters: CountComments.
 Metrics/BlockLength:
-  Max: 605
+  Max: 309
```

Comment:
Looks like this was changed unintentionally, it trigged a Rubocop error after adding lines to routes.rb https://github.com/code-dot-org/code-dot-org/pull/13775/files#r105798755